### PR TITLE
SCHED-358: Small logging improvement

### PR DESCRIPTION
**Pull request by sraaphorst**

### CODE REVIEW

File: lucupy/sky/utils.py [hunk 1]
```diff
@@ -292,8 +292,8 @@ def hour_angle_to_angle(dec: AngleParam,
     # returns hour angle at which object at dec is at altitude alt for a
     # latitude lat.
 
-    dec = np.asarray(dec.to_value(u.rad).data) * u.rad  # type: ignore
-    alt = np.asarray(alt.to_value(u.rad)) * u.rad  # type: ignore
+    dec = np.asarray(dec.to_value(u.rad).data) * u.rad
```
Comment (stroncod) [inline lucupy/sky/utils.py:295]: This should trigger mypy

File: lucupy/sky/utils.py [hunk 2]
```diff
@@ -124,7 +124,7 @@ def min_max_alt(lat: Angle, dec: AngleParam) -> Tuple[Angle, Angle]:
     return Angle(minalt, unit=u.rad), Angle(maxalt, unit=u.rad)
 
 
-def local_midnight_time(time: Time, localtzone: BaseTzInfo) -> Time:
+def local_midnight_time(time: Time, localtzone: timezone) -> Time:
```
Comment (stroncod) [inline lucupy/sky/utils.py:127]: This should trigger mypy. timezone is not a type is a function, correct type is BaseTzInfo

File: lucupy/minimodel/program.py [hunk 3]
```diff
@@ -143,7 +143,7 @@ def get_group(self, group_id: GroupID) -> Optional[Group]:
         def aux(group: Group) -> Optional[Group]:
             if group.id == group_id:
                 return group
-            elif not isinstance(group.children, Observation):
```
Comment (stroncod) [inline lucupy/minimodel/program.py:146]: This would trigger mypy

**Review by stroncod — CHANGES_REQUESTED** [review]

**Comment by stroncod** [comment]
The CD is working so there is no need to manually publish the package now.

**Comment by sraaphorst** [comment]
I'll close this PR and make a new one with the one line of code that actually is important.

File: lucupy/sky/utils.py (hunk 2 above)
Comment (sraaphorst) [reply lucupy/sky/utils.py:127]: Yes... this is weird indeed... I didn't even ask it to do this, and I did a pull before.

**Pull request closed without merging**